src/registrar/models/dns/dns_account_vendor_dns_account.py

```diff
@@ -1,9 +1,20 @@
-from django.db.models import ForeignKey, CASCADE
+from django.db import models
+from django.db.models import Q, UniqueConstraint
```

Comment:
You can also merge these two imports since ForeignKey  and CASCADE are also a part of models! That way you can reference `models.ForeignKey` as just `ForeignKey` and `models.CASCADE` as `CASCADE`


```suggestion
from django.db.models import Q, UniqueConstraint, ForeignKey, CASCADE
```